.kitchen.yml

```diff
@@ -13,12 +13,21 @@ platforms:
   - name: ubuntu-14.04
     run_list:
       - recipe[apt::default]
-
+  - name: windows-2012r2
```

Comment:
Please add the following comment above the private things:
https://github.com/chef-cookbooks/languages/blob/master/.kitchen.yml#L27-L35